doc/decision-services-arm-template-deployment.md

```diff
@@ -4,7 +4,7 @@ In this section we will be deploying Decision Services. To make this process eas
 
 If you have not done so already, please read the [prerequisites](../README.md#prerequisites) before you get started.
 
-Get the template and parameters file from the `source.zip` [here](https://github.com/InRule/AzureAppServices/releases). Both will be needed to continue with this deployment option. The steps that follow will show how to deploy using the Azure CLI, but the provided template can also be deployed through the [Azure portal](https://portal.azure.com/#create/Microsoft.Template).
+The Decision Services [ARM template](/InRule.Runtime.DecisionService.json) and [parameters file](/InRule.Runtime.DecisionService.parameters.json) can both be found in this repo. Both will be needed to continue with this deployment option. The steps that follow will show how to deploy using the Azure CLI, but the provided template can also be deployed through the [Azure portal](https://portal.azure.com/#create/Microsoft.Template).
```

Comment:
We directed people to the zip because it's versioned, unlike the direct source code link. Not sure if we considered that for this swap.